### DISCUSSION TITLE
Delete pull request with sensitive data

### DISCUSSION THREAD

**Discussion by flimagotoadmins** [community/community#24609] [Apps, API and Webhooks]
<p>Hi Guys!</p>
<p>Please, could you delete a pull request to me? This pull request contains sensitive data…</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/flimagotoadmins">@flimagotoadmins</a> and welcome to the Community Forum :stars: The Forum is unable to delete pull requests, but our Support team can evaluate your request. Contact them <a href="https://support.github.com/contact">here</a>.</p>

**Reply by Caolongg110** [reply]
> Xin chào [@flimagotoadmins](/flimagotoadmins) và chào mừng bạn đến với Diễn đàn cộng đồng🌠Diễn đàn không thể xóa các yêu cầu kéo, nhưng Nhóm hỗ trợ của chúng tôi có thể đánh giá yêu cầu của bạn. Liên hệ với họ [tại đây](https://support.github.com/contact) .
